openprescribing/frontend/management/commands/measure_definitions/ace.json

```diff
@@ -29,13 +29,145 @@
+    "bnf_code IN (",
+    "'0205051AAAAAAAA',  --Perindopril Tosilate_Tab 2.5mg \n",
+    "'0205051AAAAABAB',  --Perindopril Tosilate_Tab 5mg \n",
+    "'0205051AAAAACAC',  --Perindopril Tosilate_Tab 10mg \n",
+    "'0205051C0AAAAAA',  --Moexipril HCl_Tab 7.5mg \n",
+    "'0205051C0AAABAB',  --Moexipril HCl_Tab 15mg \n",
+    "'0205051C0BBABAB',  --Perdix_Tab 15mg \n",
+    "'0205051E0AAADAD',  --Cilazapril_Tab 2.5mg \n",
+    "'0205051E0AAAEAE',  --Cilazapril_Tab 5mg \n",
+    "'0205051E0BBABAB',  --Vascace_Tab 500mcg \n",
+    "'0205051E0BBAEAE',  --Vascace_Tab 5mg \n",
+    "'0205051F0AAAKAK',  --Captopril_Cap 2mg \n",
+    "'0205051F0AADUDU',  --Captopril_Cap 50mg \n",
+    "'0205051F0BCABAE',  --Capoten_Tab 25mg \n",
+    "'0205051I0BBAAAA',  --Innovace_Tab 2.5mg \n",
+    "'0205051I0BBABAB',  --Innovace_Tab 5mg \n",
+    "'0205051I0BBACAC',  --Innovace_Tab 10mg \n",
+    "'0205051I0BBADAD',  --Innovace_Tab 20mg \n",
+    "'0205051L0BBABAB',  --Zestril_Tab 5mg \n",
+    "'0205051L0BBACAC',  --Zestril_Tab 10mg \n",
+    "'0205051L0BBADAD',  --Zestril_Tab 20mg \n",
+    "'0205051M0BBAAAA',  --Coversyl_Tab 2mg \n",
+    "'0205051M0BBABAB',  --Coversyl_Tab 4mg \n",
+    "'0205051M0BBACAF',  --Coversyl_Tab 8mg \n",
+    "'0205051Q0BBACAC',  --Accupro_Tab 20mg \n",
+    "'0205051Q0BBADAD',  --Accupro_Tab 40mg \n",
+    "'0205051Q0BCACAC',  --Quinil_Tab 20mg \n",
+    "'0205051Q0BCADAD',  --Quinil_Tab 40mg \n",
+    "'0205051R0BBAAAA',  --Tritace_Cap 1.25mg \n",
+    "'0205051R0BBABAB',  --Tritace_Cap 2.5mg \n",
+    "'0205051R0BBADAD',  --Tritace_Cap 10mg \n",
+    "'0205051R0BBAFAK',  --Tritace_Tab 1.25mg \n",
+    "'0205051R0BBAGAL',  --Tritace_Tab 2.5mg \n",
+    "'0205051R0BBAHAM',  --Tritace_Tab 5mg \n",
+    "'0205051R0BBAIAN',  --Tritace_Tab 10mg \n",
+    "'0205051W0AAAAAA',  --Imidapril HCl_Tab 10mg \n",
+    "'0205051W0AAABAB',  --Imidapril HCl_Tab 5mg \n",
+    "'0205051W0AAACAC',  --Imidapril HCl_Tab 20mg \n",
+    "'0205051W0BBAAAA',  --Tanatril_Tab 10mg \n",
+    "'0205051W0BBABAB',  --Tanatril_Tab 5mg \n",
+    "'0205051W0BBACAC',  --Tanatril_Tab 20mg \n",
+    "'0205051Y0AAAAAA',  --Perindopril Arginine_Tab 2.5mg \n",
+    "'0205051Y0AAABAB',  --Perindopril Arginine_Tab 5mg \n",
+    "'0205051Y0AAACAC',  --Perindopril Arginine_Tab 10mg \n",
+    "'0205051Y0BBAAAA',  --Coversyl Arginine_Tab 2.5mg \n",
+    "'0205051Y0BBABAB',  --Coversyl Arginine_Tab 5mg \n",
+    "'0205051Y0BBACAC'  --Coversyl Arginine_Tab 10mg)"
```

Comment:
This closing paren is commented out. It needs to be on its own line

Reply:
fixed